.mockery.yaml

```diff
@@ -17,6 +17,10 @@ packages:
       OffRampReader:
         config:
           filename: offramp_reader_mock.go
+  github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3:
```

Comment:
If you aren't using these here, you can mock them in your other repo.

Reply:
Removed